packaging/install.cmake

```diff
@@ -92,10 +92,13 @@ if(UNIX)
 
   # replace MPIEXEC MPIEXEC_NUMPROC_FLAG and MPI_IMPL KHIOPS_MPI_EXTRA_FLAG ADDITIONAL_ENV_VAR
   if("${MPI_IMPL}" STREQUAL "openmpi")
-    set(KHIOPS_MPI_EXTRA_FLAG "--allow-run-as-root --quiet")
+    set(KHIOPS_MPI_EXTRA_FLAG "--allow-run-as-root")
+    set(KHIOPS_MPI_QUIET "--quiet")
     set(ADDITIONAL_ENV_VAR "export OMPI_MCA_btl_vader_single_copy_mechanism=none # issue on docker")
+    set(ADDITIONAL_ENV_VAR_DISPLAY "\techo OMPI_MCA_btl_vader_single_copy_mechanism none")
     if(IS_FEDORA_LIKE)
       set(ADDITIONAL_ENV_VAR "${ADDITIONAL_ENV_VAR}\nexport PSM3_DEVICES=self # issue on rocky linux")
+      set(ADDITIONAL_ENV_VAR_DISPLAY "${ADDITIONAL_ENV_VAR_DISPLAY}\n\techo PSM3_DEVICES self")
```

Comment:
Why not `echo PSM3_DEVICES "$PSM3_DEVICES"` to be consistent with the other variables?

Reply:
yes why not...